fetchmail_notify_error_to_sender/data/email_template_data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+    <!--Email template -->
+    <record id="email_template_error_notice" model="mail.template">
+        <field name="name">Fetchmail - error notice</field>
+        <field name="email_from">${ctx.get('sender_message').get('to')|safe}</field>
+        <field name="email_to">${ctx.get('sender_message').get('from')|safe}</field>
+        <field name="subject">Receiving error with: ${ctx.get('sender_message').get('subject')|safe}</field>
+        <field name="model_id" ref="fetchmail.model_fetchmail_server"/>
+        <field name="auto_delete" eval="True"/>
+        <field name="user_signature" eval="True"/>
+        <field name="lang">${ctx.get('lang')}</field>
+        <field name="body_html"><![CDATA[
+            <div>
```

Comment:
You changed the indentation of the body_html contents. I suspect that this invalidates existing translations because they are whitespace sensitive.